### PR TITLE
[ticket/16810] Fix No end tag in Custom BBCodes

### PR DESCRIPTION
**Pull request by 3D-I**
PHPBB3-16810

Checklist:

- [x] Correct branch: master for new features; 3.3.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html) and [3.3.x](https://area51.phpbb.com/docs/dev/3.3.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.3.x/development/git.html)

https://tracker.phpbb.com/browse/PHPBB3-16810

### CODE REVIEW

File: phpBB/includes/functions_display.php [hunk 1]
```diff
@@ -1119,13 +1119,18 @@ function display_custom_bbcodes()
 		// Convert Numeric Character References to UTF-8 chars.
 		$row['bbcode_helpline'] = utf8_decode_ncr($row['bbcode_helpline']);
 
-		$custom_tags = array(
-			'BBCODE_NAME'		=> "'[{$row['bbcode_tag']}]', '[/" . str_replace('=', '', $row['bbcode_tag']) . "]'",
+		// Does the closing bbcode tag exists? If so display it.
+		$bbcode_close_tag	= '%\[\/' . utf8_strtolower($row['bbcode_tag']) . ']%';
```
Comment (CHItA) [inline phpBB/includes/functions_display.php:1123]: doesn't `]` need to be escaped?

Comment (3D-I) [reply phpBB/includes/functions_display.php:1123]: IMHO is not necessary.
Test; https://regex101.com/r/NxgOZv/1

Comment (marc1706) [reply phpBB/includes/functions_display.php:1123]: I would still opt for escaping it to have it uniform.

**Comment by CHItA** [comment]
@marc1706 it seems commit message validation is broken again. I believe we don't allow merge commits in PRs, right?

**Comment by marc1706** [comment]
I think we've made it less sensitive to that issue but I agree that merge commits shouldn't be in PRs.

**Pull request closed without merging**